guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -5,6 +5,8 @@
 To configure an external DHCP server to use with {ProductName}, on a Red{nbsp}Hat Enterprise Linux server, you must install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) packages.
 You must also share the DHCP configuration and lease files with {ProductName}. The example in this procedure uses the distributed Network File System (NFS) protocol to share the DHCP configuration and lease files.
 
+NOTE: If you use dnsmasq as an external DHCP server, enable the `dhcp-no-override` setting. This is required because {Project} creates configuration files on the TFTP server under the `grub2/` subdirectory. While with the `dhcp-no-override` setting disabled, clients fetch the bootloader and its configuration from the root directory, which might cause an error.
```

Comment:
```suggestion
NOTE: If you use dnsmasq as an external DHCP server, enable the `dhcp-no-override` setting. This is required because {Project} creates configuration files on the TFTP server under the `grub2/` subdirectory. If the `dhcp-no-override` setting is disabled, clients fetch the bootloader and its configuration from the root directory, which might cause an error.
```